packages/Main/src/Layer/PointCloudLayer.js

```diff
@@ -287,35 +344,39 @@ class PointCloudLayer extends GeometryLayer {
             return;
         }
 
-        // pick the best bounding box
-        const bbox = (elt.tightbbox ? elt.tightbbox : elt.bbox);
-        elt.visible = context.camera.isBox3Visible(bbox, this.object3d.matrixWorld);
+        // get object on which to measure distance
+        let bbox;
+        let matrixWorld;
+        let matrixWorldInverse;
+        if (elt.obj) {
+            const obj = elt.obj;
+            bbox = obj.geometry.boundingBox;
+            matrixWorld = obj.matrixWorld;
+            matrixWorldInverse = obj.matrixWorldInverse;
+        } else {
+            bbox = elt.clampOBB.box3D;
+            matrixWorld = elt.clampOBB.matrixWorld;
+            matrixWorldInverse = elt.clampOBB.matrixWorldInverse;
+        }
```

Comment:
could be simplify ?

Reply:
verify the transformation between elt, elt.obj, and and clampOBB?

Reply:
```js
let bbox;
let object3d;
if (elt.obj) {
    object3d = elt.obj;
    bbox = obj.geometry.boundingBox;
} else {
    object3d = elt.clampOBB;
    bbox = object3d.box3D;
}

const { matrixWorld, matrixWorldInverse } = object3d;
```

Reply:
Done